packages/fxa-payments-server/src/routes/Product/SubscriptionCreate/index.scss

```diff
@@ -6,32 +6,32 @@
   grid-row: 2/3;
   border-top: 0;
   border-radius: 0 0 8px 8px;
-  box-shadow: 0 -1px white, 0 1px white, -2px 2px 2px -2px rgba(12, 12, 13, 0.1),
-    2px 2px 2px -2px rgba(12, 12, 13, 0.1);
+  box-shadow: 0 -1px white, 0 1px white, -2px 2px 2px -2px rgba(158, 158, 158, 1),
+    2px 2px 2px -2px rgba(158, 158, 158, 1);
 
   @include max-width('tablet') {
     grid-row: 4/5;
-    box-shadow: 0px 1px 4px 0px rgba(12, 12, 13, 0.1);
+    box-shadow: 0px 1px 4px 0px rgba(158, 158, 158, 1);
     border-radius: 8px;
   }
 
   @media (orientation: landscape) and (max-width: 927px) and (max-height: 429px) {
     grid-row: 4/5;
-    box-shadow: 0px 1px 4px 0px rgba(12, 12, 13, 0.1);
+    box-shadow: 0px 1px 4px 0px rgba(158, 158, 158, 1);
     border-radius: 8px;
   }
 
   @media (orientation: portrait) and (max-width: 429px) and (max-height: 927px) {
     grid-row: 4/5;
-    box-shadow: 0px 1px 4px 0px rgba(12, 12, 13, 0.1);
+    box-shadow: 0px 1px 4px 0px rgba(158, 158, 158, 1);
     border-radius: 8px;
   }
 }
 
 .product-payment {
```

Comment:
I can't select some of the lines so I'll address all suggested changes to reduce DRY in this selector here. Bear with me and let me know if you have any questions:
1. Remove both media queries from **L45-L55**.
2. Move **L33-L37** below **L42**.
3. Remove **L39** and **L43** (now **L48** after adding the lines from (2), I believe).
4. Add two declarations below **L32**: `min-height: auto;` and `margin: auto;`.
5. Wrap **L32** and the two new declarations from (4) in a `@include min-width('tablet')`